### DISCUSSION TITLE
v1.6.0

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#230] [Release]
<div>
    <a><img src="https://user-images.githubusercontent.com/16687318/156849195-06d5c58f-d472-45a7-9ec4-26abc72bc27f.png" width="49%"></a>
    <a><img src="https://user-images.githubusercontent.com/16687318/154815173-454084a4-3b88-48ca-b62b-6df09fe853ff.png" width="49%"></a>
</div>

**Features**
Update private match map select menu ([#194](https://github.com/R2Northstar/NorthstarLauncher/pull/194))
Implemented stub loading which greatly reduces memory usage when running dedicated servers ([#81](https://github.com/R2Northstar/NorthstarLauncher/pull/81))
Added 'Respawn Protection' match setting ([#167](https://github.com/R2Northstar/NorthstarMods/pull/167))
New chathooks available: server broadcasts, whispers and more. ([#74](https://github.com/R2Northstar/NorthstarLauncher/pull/74)) and ([#217](https://github.com/R2Northstar/NorthstarMods/pull/217)). Examples [here](https://r2northstar.readthedocs.io/en/latest/reference/northstar/chathooks.html?highlight=chat#custom-messages)
Playlist overrides now work on listen servers ([#70](https://github.com/R2Northstar/NorthstarLauncher/pull/70))
Updated list of Gun Game weapons ([#252](https://github.com/R2Northstar/NorthstarMods/pull/252))
Auto-generate startup arg files ([#99](https://github.com/R2Northstar/NorthstarLauncher/pull/99))
ConVar class rebuild ([#90](https://github.com/R2Northstar/NorthstarLauncher/pull/90))
Added function to update vortex shield ignored weapons ([#250](https://github.com/R2Northstar/NorthstarMods/pull/250))
Added a default plugin for Discord Rich Presence integration. ([NorthstarDiscordRPC](https://github.com/R2Northstar/NorthstarDiscordRPC))

**Bugfixes**
Update floor is lava riff on maps Crash Site, Drydock, and Complex ([#181](https://github.com/R2Northstar/NorthstarMods/pull/181))
Fixed lethal fog on Deck and Stacks ([#181](https://github.com/R2Northstar/NorthstarMods/pull/181))
Amped Hardpoint now works on Homestead ([#174](https://github.com/R2Northstar/NorthstarMods/pull/174))
Fixed hardpoints getting stuck on amped status ([#199](https://github.com/R2Northstar/NorthstarMods/pull/199))
Implemented amped/unamped hardpoint prop animations
Fixed Last Titan Stading battery spawns on Angel City and Forwardbase Kodai ([#224](https://github.com/R2Northstar/NorthstarMods/pull/224))
Core meter now shows progression towards electric smoke on titans ([#212](https://github.com/R2Northstar/NorthstarMods/pull/212))
Multiple resolutions for the Northstar icon ([#97](https://github.com/R2Northstar/NorthstarLauncher/pull/97))
Fix Game Crash on Amped Hardpoint when player not in Capper Array [#279](https://github.com/R2Northstar/NorthstarMods/pull/279)
Updated the localization files

**Backend**
We now have a landing page ([#32](https://github.com/R2Northstar/NorthstarMasterServer/pull/32)), check https://northstar.tf
Added caching of server list ([#30](https://github.com/R2Northstar/NorthstarMasterServer/pull/30))
Added version numbering for master server requests ([#113](https://github.com/R2Northstar/NorthstarLauncher/pull/113))
Add check for player name correctness. ([#86](https://github.com/R2Northstar/NorthstarLauncher/pull/86))
The build pipelines now generate a `md5sum.txt` file, this can be used to ensure that the downloaded release matches with the git repository.

**Contributors:**
@abarichello @ASpoonPlaysGames @barnabwhy @begin-theadventure @BobTheBob9 @cpdt @Dinorush @emma-miler @F1F7Y @fvnkhead @GeckoEidechse @H0L0theBard @hummusbird @JMM889901 @KittenPopo @lapaxx @laundmo @Mauler125 @MysteriousRSA @Neoministein @Orpheus2401 @PANCHO7532B @pg9182 @RoyalBlue1 @SamLam140330 @taskinoz @uniboi @x3Karma @xamionex

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.6.0'>v1.6.0</a>.</em>

**Comment by xamionex** [comment]
![](https://cdn.discordapp.com/emojis/925417656146878465.webp)

**Comment by Soreo** [comment]
I can't play Titanfall 2 with Northstar 1.6.0. I started Origin. I click on NorthstarLauncher and nothing happens. The terminal opens and that's it. Before Northstar 1.6.0, works fine.

P.S.: Used by Google Translate.

**Reply by F1F7Y** [reply]
delete the `plugins` folder that's located in `Titanfall2/R2Northstar/`. If it still crashes make a ticket on discord or comment on this thread

**Reply by Soreo** [reply]
I deleted the folder "plugins" and Titanfall 2 started. It is working. Thank you for help.
